src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2390,6 +2390,24 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             $container->removeDefinition('notifier.channel.email');
         }
 
+        $servicesWithBusArgument = ['texter', 'chatter', 'notifier.channel.chat', 'notifier.channel.email', 'notifier.channel.sms'];
+        if (false === $messageBus = $config['message_bus']) {
```

Comment:
should we throw an exception if one tries to configure `message_bus` while messenger is not installed/enabled?

Reply:
Yes, makes sense to me. Will do that.

Reply:
- [ ] throw exception if messenger is not installed